src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -776,9 +776,25 @@ The accounting information can be accessed only through a *JMX client* or the Pr
 [[_run_as_me]]
 == Run Computation with a user's system account
 
-When starting a ProActive Node, it can be configured to run tasks under the user's system account when
-link:ProActiveUserGuide.html#_run_computation_with_your_system_account[runAsMe] is set to true in
-the job's definition. The default is to use the password method.
+Configure a ProActive Node to execute tasks under a user's system account by ticking the
+link:ProActiveUserGuide.html#_run_computation_with_your_system_account[Run as me] box in the task configuration.
```

Comment:
Yes, for sure it works. My question was: why to redirect to _Run as me_ section since we are in _Run as me_  section when reading this part...